### ISSUE TITLE
Login avatar not shown on beautifulSoup docs

### ISSUE THREAD

**Issue by rbbeeston** [boostorg/website-v2#1307]
When logged in to the site, going to a library item documentation the "Join" text is shown instead of the avatar.

**Comment by sdarwin** [comment]
- This had been working correctly before.
- An avatar still appears on production. boost.io.
- Specifically with doc pages, the avatar (and maybe profile info) are retrieved from an API, rather than in the usual way, which allows CDN caching of the page.

**Comment by rbbeeston** [comment]
yes, it had been working before. Dave did make some changes earlier that might have affected it. 
Those changes aren't in master currently, so it wouldn't be seen there.

**Comment by sdarwin** [comment]
In the initial report:  
> going to a library item documentation

Please include an example link of where exactly the problem was occurring, to be able to verify.

Most likely this is fixed. There was a merged PR with the title "Resolve issue with avatars on docs/guide page".     And docs display have gone through multiple iterations since then, also.     Not being able to replicate the issue, and with that PR completed, I'll close the issue. Re-open or create a new github issue if something similar happens.
